src/content/docs/three-js/three-js-case-study.md

```diff
@@ -0,0 +1,148 @@
+
+| Scenes Graph | Two spatial objects rotate around each other, mimicking the Sun-Earth system, showcasing how scene graphs can represent complex spatial relationships |
+https://threejs.org/manual/examples/scenegraph-sun-earth.html
+
+| Camera Controls | Two objects placed at the center scene with ability to zoom in and out, demonstrating how camera perspective controls work in 3D space.|
+https://threejs.org/manual/examples/cameras-perspective.html
+
+| Renderers | Example of rendering animation using skinning and blending, showing how renders handle different animation techniques in Three.js. |
+https://threejs.org/examples/#webgo_animation_skinning_blending
+
+| Geometry | Six geometric objects levitate in synchronized pace and direction demonstrating various types of parametric geometry available in Three.js. |
+https://threejs.org/examples/#webgl_geometries_parametric
+
+| Materials | A large object is divided into different colors which change at the set pace. The texture also changes, interacting with light to demonstrate the dynamic capabilities of materials. |
+https://threejs.org/examples/#webgl_framebuffer_texture
+
+| Animation System | This example represents an animation system using dynamics, instancing, showing various movements and interactions in 3D space.|
+https://threejs.org/examples/#webgl_instancing_dynamic
+
+| Lights and Shadows| An object creates shadows as it interacts with lights and moves demonstrating the effects of lighting and shadows in a 3D environement. |
+https://threejs.org/examples/#webgl_geometry_csg
+
+[medium.com] [codeparrot.ai] [threejs.org]
+
+## Market Comparison
+
+While Three.js initially appears to be distinctive, there exist several alternative frameworks for JavaScript and web development that integrate 3D models to create interactive and visually appealing designs.
+
+Some of these frameworks have smaller communities and are still in the early stages of development. For instance, Whitestorm.js is a newer framework that incorporates the latest ECMAScript features, but it hasn’t gained a huge following yet [slant.co].
+
+While only having a small community, Babylon.js, is one of the main competitors to Three.js and offers support for Apple's AR goggles. Babylon.js presents a more professional appeal and delivers visual fidelity that surpasses both Three.js and Whitestorm [slant.co] [babylonjs.com].
+
+PlayCanvas, a third competitor, claims to have excellent cross-platform compatibility. It boasts similar features as both Three.js and Babylon.js. It provides users with many tools geared towards game development, although some of its features are hidden behind a paid subscription [slant.co] [playcanvas.com].
+
+One of Three.js’ main strengths appears to be the great documentation provided by the developers and also its large number of official plugins and tutorials [slant.co].
+
+###The following table compares Three.js with its main competitors across various key points:
```

Comment:
Should this be a heading?

Reply:
I have now deleted the subheading , making it a part of the text instead.